common/models/user.js

```diff
@@ -38,6 +42,21 @@ module.exports = function(User) {
   // see:
   // https://github.com/strongloop/loopback/issues/1137#issuecomment-109200135
   delete User.validations.email;
+  // Updated workaround for email validation
+  // see:
+  // https://github.com/strongloop/loopback/issues/1137#issuecomment-271437926
+  User.validate('email', function( err ) {
+    if (this.email && !isEmail(this.email)) {
+      debug(err);
```

Comment:
`err` here is a callback, it indicates whether the validation has failed when called.

see: http://apidocs.strongloop.com/loopback-datasource-juggler/#validatable-validate